src/overtone/examples/midi/keyboard.clj

```diff
@@ -85,9 +86,18 @@
                       ; sound).
                       :finished {}})
         sustain* (atom 0)
+        pitch-bend* (atom 0)
         on-id (keyword (gensym "on-handler"))
         off-id (keyword (gensym "off-handler"))
-        cc-id (keyword (gensym "cc-handler"))]
+        cc-id (keyword (gensym "cc-handler"))
+        pitch-bend-id (keyword (gensym "pitch-bend-handler"))
+        bytes-to-short (fn [b1 b2]
+                         (let [bb (. ByteBuffer allocate 2)
```

Comment:
This is a workaround as I'm not good in bit operations. I'd be more glad with a cleaner solution (no Java).